dbfit-java/build.gradle

```diff
@@ -222,3 +251,116 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
```

Comment:
I wonder if this can be somehow automatically collected from the projects' dependencies definition (if it's possible to `tag` public dependencies there). Just an idea, I'm not sure it's feasible/easy to achieve.